tests/playwright/tests/utils/general.ts

```diff
@@ -38,7 +38,7 @@ function isSubset(obj, potentialSubset) {
           return false;
         }
         // eslint-disable-next-line no-prototype-builtins
-      } else if (!obj.hasOwnProperty(prop) || objValue !== potentialValue) {
+      } else if (!obj.hasOwnProperty(prop) || objValue != potentialValue) {
```

Comment:
shouldn't be `!==`?

Reply:
Strict type check is disabled since dbs are returning number data in number/string type. Once we standardise the response type we can change it to `!==`

Reply:
got it. suggest to add this comment in code as well.